packages/arb-token-bridge-ui/src/components/TransactionHistory/helpers.ts

```diff
@@ -570,5 +570,14 @@ export function getTxHumanReadableRemainingTime(tx: MergedTransaction) {
   if (minutesLeft > 0) {
     return formattedMinutesLeft
   }
-  return 'Almost there...'
+  return 'less than a minute'
+}
+
+export function getDestinationNetworkTxId(tx: MergedTransaction) {
+  if (tx.isCctp) {
+    return tx.cctpData?.receiveMessageTransactionHash
+  }
+  return tx.isWithdrawal
+    ? tx.l2ToL1MsgData?.uniqueId.toString()
```

Comment:
Wouldn't this `uniqueId` already be string?

Reply:
It's a BigNumber